pypeman/contrib/http.py

```diff
@@ -24,7 +24,7 @@ def __init__(
             loop=None,
             http_args=None,
             host=None,
-            sock=None,
+            sock=None,  # TODO: Why have sock and port if it's the same ?
```

Comment:
you might add a deprecation warning if hosts / host+port are used and indicate, that only sock shall be used
and indicate a version (e.g. 0.7 where support will be removed)

Reply:
Mmm, yeah i can do this, but not in this mr

Reply:
I opened an issue